src/truststore/_api.py

```diff
@@ -273,6 +272,23 @@ def verify_mode(self, value: ssl.VerifyMode) -> None:
         )
 
 
+# Python 3.13+ makes get_unverified_chain() a public API that only returns DER
+# encoded certificates. We detect whether we need to call public_bytes() for 3.10->3.12
+# Pre-3.13 returned None instead of an empty list from get_unverified_chain()
+if sys.version_info >= (3, 13):
+
+    def _get_unverified_chain_bytes(sslobj: ssl.SSLObject) -> list[bytes]:
+        unverified_chain = sslobj.get_unverified_chain() or ()  # type: ignore[attr-defined]
```

Comment:
This is never `None` on 3.13: https://github.com/python/cpython/blob/57aee2a02ce38be30cf6c310245547cf56562ab6/Lib/ssl.py#L903-L912

Reply:
Thanks for reviewing! That's good to know, I think I'll keep the code as-is since it's not getting in the way of functionality.

Reply:
Just thought I'd point it out :)